src/pyhf/tensor/jax_backend.py

```diff
@@ -179,14 +179,14 @@ def conditional(self, predicate, true_callable, false_callable):
 
     def tolist(self, tensor_in):
         try:
-            return onp.asarray(tensor_in).tolist()
+            return np.asarray(tensor_in).tolist()
```

Comment:
`jax.numpy` now supports `tolist` so this could instead be

```suggestion
            return jnp.asarray(tensor_in).tolist()
```

but this should probably be left to another follow up PR as this is a style PR where no real code is supposed to change.

Reply:
#1137